Adds redirects for common inbound links to hit 404 pages.

johnnymatthews opened the pull request:
Redirects some [common inbound links that hit 404 pages](https://datastudio.google.com/reporting/62116ab2-5024-4e38-b35e-91b1a1e85806/page/pP5aB?utm_source=google-datastudio&utm_medium=email&utm_campaign=scheduled-report).

cwaring commented:
@johnnymatthews we use the [_redirects](https://github.com/filecoin-project/filecoin-docs/blob/master/docs/.vuepress/public/_redirects) file in this repo until we move across to fleek, any items should be added there. I'm also evaluating the support of the new cloudflare page redirects so we may be able to remove meta redirects in the future!

https://blog.cloudflare.com/cloudflare-pages-ga/

johnnymatthews commented:
> we use the _redirects file in this repo - @cwaring 

That'll explain why the redirects I added weren't doing anything. Moved them into the `_redirects` file now.
